src/main/java/TaskType.java

```diff
@@ -0,0 +1,3 @@
+public enum TaskType{
+    TODO, DEADLINE, EVENT
```

Comment:
Enums that represent specific task types, useful for further addition of other task types in the future. Adheres to coding standard.